orcid-core/src/main/java/org/orcid/core/manager/impl/OrcidSecurityManagerImpl.java

```diff
@@ -292,12 +292,14 @@ private OAuth2Authentication getOAuth2Authentication() {
 
     @Override
     public String getClientIdFromAPIRequest() {
-        OAuth2Authentication oAuth2Authentication = getOAuth2Authentication();
-        if (oAuth2Authentication == null) {
-            return null;
+        SecurityContext context = SecurityContextHolder.getContext();
```

Comment:
This method is used to get the client id from the authentication, we are using it in the OrcidExceptionMapper to log the client ID that is doing the request.

But, for the internal API this is not a OAuth2Authentication, so, when we call the method getOAuth2Authentication() it throw an "AccessControlException", and, that prevent us to log the error that is happening.